web/src/main/java/org/openmrs/web/filter/util/FilterUtil.java

```diff
@@ -28,6 +28,14 @@
  */
 public class FilterUtil {
 	
+	/*
+	Utility classes should not have public constructors.
+        Utility classes, which are a collection of static members, are not meant to be instantiated.
+	*/
```

Comment:
I think we don't need this doc comment, since it is self-explanatory.

Reply:
Removed it in the new PR sir !!